lib/puppet/provider/package/pacman.rb

```diff
@@ -28,7 +28,9 @@ def self.yaourt?
   # Checks if a given name is a group
   def self.group?(name)
     begin
-      !pacman("-Sg", name).empty?
+      !execpipe([command(:pacman), "-Sg"], false).empty? do |process|
```

Comment:
This is applying `!` to the string, which will always be false.

We need to add a `.empty?` to the closing `end` of the block.